pkg/repository/iwatch.go

```diff
@@ -23,14 +23,14 @@ import (
 	"time"
 
 	"github.com/go-logr/logr"
+	"github.com/kubebb/core/api/v1alpha1"
```

Comment:
GCI, a tool that control golang package import order and make it always deterministic.

you can run `golangci-lint run --fix -v ./...` locally to fix this issue.